src/training/flava_eval/finetune_glue.py

```diff
@@ -163,29 +202,39 @@ def get_task_metric(task_name):
 def get_task_dataloaders(args):
     tokenizer = get_tokenizer(args.model)
     task_name = args.task_name
+    roberta_tokenizer = isinstance(tokenizer, transformers.models.roberta.tokenization_roberta.RobertaTokenizer) or isinstance(tokenizer, transformers.models.roberta.tokenization_roberta_fast.RobertaTokenizerFast)
+    separator_token = '</s>' if roberta_tokenizer else '<end_of_text>'
```

Comment:
For HF tokenizers does `tokenizer.special_tokens_map` not work?

Reply:
not always, if the tokenizer doesn't have a sep token specified it won't default to the eos token